### ISSUE TITLE
[Bug Report] New Version Does Not Do a Full Install

### ISSUE THREAD

**Issue by john-okeefe** [dragoonDorise/EmuDeck#1308]
### Prerequisite Steps

- [ ] I have searched existing issues
- [ ] This issue is not a duplicate of an existing one
- [ ] I have checked the [EmuDeck Wiki](https://emudeck.github.io/)
- [ ] I have read the [Troubleshooting Page](https://emudeck.github.io/troubleshooting/steamos/) on the EmuDeck Wiki

### What distro are you running?

Arch Linux

### Which SteamOS Branch are you using?

Stable

### What happened?

This impacts both version 2.2.18 and EmuDeck EA Stable.

I've tried several times to do a clean install on Arch Linux (both my laptop and a clean VM install).

Reproduction Steps: Select Easy Mode (I tried custom too, same results), Internal Storage, Select Linux PC (tried selecting SteamDeck as well). 

I've tried several installs and clean setups. I've tried with Plasma, XFCE, LXQT.

When the install  and setup finish, I'm left with the screenshot below.

The only way I  was able to install it fully on my Laptop was using the AUR package (version 2.1.18) and then upgrade to the EmuDeck-EA.

![Screenshot_20240826_204824](https://github.com/user-attachments/assets/dc6353be-f113-423c-93d9-e21c7ef55052)
[emudeckSetup.log](https://github.com/user-attachments/files/16802946/emudeckSetup.log)

**Comment by dragoonDorise** [comment]
@john-okeefe still having this issue?

**Comment by john-okeefe** [comment]
@dragoonDorise No. Now it's worse. It throws Javascript exceptions and crashes at each install step and doesn't install any of the Emulators.
[emudeck-term.log](https://github.com/user-attachments/files/17483477/emudeck-term.log)

**Comment by john-okeefe** [comment]
Repro Steps (for Arch VM): Install Arch using archinstall. Make sure you select Desktop and pick any environment. Make sure to install a web browser. Go to EmuDeck site, download the Linux link and run with 'sh Emudeck.sh'.

**Comment by tweelix** [comment]
I'm having the same issue, also on a fresh Arch Linux install, not SteamOS
